client/src/store/modules/scheduledEvent/normalizer.js

```diff
@@ -29,16 +29,11 @@ export const eventEmailMapper = Email => ({
 });
 
 export const eventApiMapper = Event => ({
-    id: Event.id,
```

Comment:
seems scary, nothing will break?

@yevhen-turovets

Reply:
It might be for @aussie-m-mike

Reply:
No, it is my code. I think this is a break code. Why you deleted them?

Reply:
I thought that we don't need it.

Reply:
do these properties lead to incorrect behavior? If not, return them back, because @yevhen-turovets is working on this right now, and your commits may break his logic